_posts/2018-01-02-this-is-not-a-test.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  "This Is Not a Test"
+date:   2018-01-02 12:00:00 -0400
+categories: culture interviews
+author: Janice Smith
+---
+
+At forward financing how we work is something we are always trying to improve on.
+We have some core tenants to how we work as a team, our director of engineering,
+Patrick Hereford, wrote a great post about that recently. You should check it out [here](http://tech.forwardfinancing.com/engineering/culture/2017/09/30/our-engineering-culture-methodologies-and-philosophies.html){:target='_blank'}
+
+We have been in the middle of building the team’s numbers here,
+in a push that will eventually more than double the size or our group.
```

Comment:
`of our group`

Reply:
"building the team’s numbers here" => This reads awkward. We've been hiring to meet the aggressive growth demands of the company. Putting that in context would also make us look good.

Reply:
💯